ui/admin/app/styles/app.scss

```diff
@@ -630,11 +630,15 @@
     min-height: 4rem; // 1
   }
 
+  // these styles are not applied directly to the order-series-diagram
+  // because these are unique to the target form and may not be
+  // needed in other instances of the diagram
   .worker-wrapper {
-    display: flex;
-    align-items: center;
-    min-height: 11rem;
-    margin-top: sizing.rems(xs) - sizing.rems(xxxxs); // 10
+    .ordered-series-diagram {
+      align-items: center;
```

Comment:
Why is this necessary?

Reply:
With `align-items`
<img width="620" alt="Screen Shot 2023-02-08 at 12 39 31 PM" src="https://user-images.githubusercontent.com/29386339/217622213-9ebf2a99-3853-446e-97e5-2054e1d1d6ef.png">
without `align-items`
<img width="620" alt="Screen Shot 2023-02-08 at 12 39 42 PM" src="https://user-images.githubusercontent.com/29386339/217622333-4d80f217-cea4-478c-88bd-33bad48f07f9.png">

It keeps the diagram centered vertically to complement the min hight set.

Reply:
Moved style changes out of this PR.